models/app-models.js

```diff
@@ -39,4 +39,19 @@ exports.selectArticleById = async (article_id) => {
     return Promise.all(querProms).then((result) => {
                 return result[querProms.length - 1].rows
             })
+}
+
+exports.selectCommentsByArticle = async (article_id) => {
+    let artiQuery = "SELECT * FROM comments"
+    let queryVal = []
+    let querProms = []
```

Comment:
Do these need to be let? we are never reassigning their values, just modifying the contents of them.